docs/src/whatsnew/latest.rst.template

```diff
@@ -56,6 +56,10 @@ NOTE: section above is a template for bugfix patches
 
 #. N/A
```

Comment:
Need to get rid of this line!

Reply:
Decided to edit the right file instead...